### ISSUE TITLE
aria-describedby attribute

### ISSUE THREAD

**Issue by yantei** [colinaut/alpinejs-plugin-simple-validate#34]
Please add aria-describedby attribute to elements with validation messages if the original input has an ID.
Now your solution fails accessibility checks and we need to write our own code to add the attribute.

**Comment by colinaut** [comment]
Thanks for the heads up! I'll work on adding this next week

**Comment by yantei** [comment]
One update - aria-describedby attribute should be added to inputs, containing IDs of live appended validation elements (not reverse)
